irohad/consensus/consensus_service.hpp

```diff
@@ -20,13 +20,15 @@
 
 #include <consensus.grpc.pb.h>
 #include <consensus/messages.hpp>
+#include <uvw/emitter.hpp>
 
-namespace iroha {
+namespace consensus {
 
   using ServerContext = grpc::ServerContext;
   using Status = grpc::Status;
```

Comment:
Probably this is not okay. I will change it in another branch. I pushed it in this branch accidentally. Sorry :)